src/graphql/resolvers/getOffersResolver.test.js

```diff
@@ -1,40 +1,88 @@
+import GetOffersResolver from "./getOffersResolver";
+
 describe("Resolvers", () => {
   it("GetOffersResolver", async () => {});
 });
 
-// const testcases = [
-//   {
-//     desc: "should return student object with the given args and a uuid",
-//     db: {
-//       put: jest.fn().mockReturnValue({ promise: () => {} })
-//     },
-//     args: {
-//       student: {
-//         firstname: "Braden",
-//         lastname: "Watkins"
-//       }
-//     },
-//     expectedDBCall: {
-//       TableName: "Student",
-//       Item: {
-//         firstname: "Braden",
-//         lastname: "Watkins",
-//         id: uuidv4()
-//       }
-//     },
-//     expectedRetValue: {
-//       firstname: "Braden",
-//       lastname: "Watkins",
-//       id: uuidv4()
-//     }
-//   }
-// ];
+const offers = [
+  {
+    offer_id: "1234",
+    position_type: "Full-time",
+    position_title: "Janitor",
+    accepted: true,
+    academic_year: "Senior",
+    company_name: "Qualtrics",
+    flag: true,
+    student_id: "My student id",
+    location: {
+      city: "Provo",
+      state: "Utah",
+      country: "USA"
+    },
+    wage_value: 123456,
+    wage_type: "salary"
+  }
+];
+
+const bonuses = [
+  {
+    value: 20000,
+    type: "Free money",
+    repeat_count: 5,
+    one_time: false,
+    description: "Just some free money for the new janitor"
+  }
+];
+
+const testcases = [
+  {
+    desc: "should return all offer objects",
+    db: {
+      scan: jest.fn().mockReturnValue({
+        promise: () => {
+          return {
+            Items: offers
+          };
+        }
+      }),
+      query: jest.fn().mockReturnValue({
+        promise: () => {
+          return {
+            Items: bonuses
+          };
+        }
+      })
+    },
+    expectedDBCalls: {
+      scan: { TableName: "Offer" },
+      query: {
+        TableName: "Bonus",
+        KeyConditionExpression: "#i = :id",
+        ExpressionAttributeNames: { "#i": "id" },
+        ExpressionAttributeValues: {
+          ":id": "1234"
+        }
+      }
+    },
+    expectedRetValue: {
+      edges: [
+        {
+          ...offers[0],
+          bonuses: bonuses
+        }
+      ]
+    }
+  }
+];
 
-// testcases.forEach(
-//   async ({ db, args, expectedDBCall, expectedRetValue }, i) => {
-//     await expect(PostStudentResolver(db, args)).resolves.toEqual(
-//       expectedRetValue
-//     );
-//     expect(db.put.mock.calls[i][0]).toEqual(expectedDBCall);
-//   }
-// );
+testcases.forEach(
```

Comment:
jest won't run these because they're not in a test suite:
```
describe('Test suite name', () => {
  it('Test name', () => {
    // test cases here
  })
})
```

Reply:
🤦‍♂️